File: app/src/androidTest/java/net/tlalka/android/fiszki/view/pages/MainPage.java
package net.tlalka.android.fiszki.view.pages;

import net.tlalka.android.fiszki.R;

public class MainPage extends AbstractPage {

    @Override
    public void valid() {
        isVisible(R.id.main_list_view);
        isImageDescription(R.string.app_name);
    }

    public void openLessons() {
        clickListItem(R.id.main_list_view, 0);
    }

    public void openTests() {
        clickListItem(R.id.main_list_view, 1);
    }

    public void openWords() {
        clickListItem(R.id.main_list_view, 2);
    }

    public void openHelp() {
        clickListItem(R.id.main_list_view, 3);
    }
}

File: app/src/androidTest/kotlin/net/tlalka/fiszki/view/pages/MainPage.kt
package net.tlalka.fiszki.view.pages

import net.tlalka.fiszki.R

class MainPage : AbstractPage() {

    override fun valid() {
        isVisible(R.id.main_list_view)
        isImageDescription(R.string.app_name)
    }

    fun openLessons() {
        clickListItem(R.id.main_list_view, 0)
    }

    fun openTests() {
        clickListItem(R.id.main_list_view, 1)
    }

    fun openWords() {
        clickListItem(R.id.main_list_view, 2)
    }

    fun openHelp() {
        clickListItem(R.id.main_list_view, 3)
    }
}
